plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/ParquetTester.java

```diff
@@ -478,7 +479,7 @@ private void assertFileContents(
             List<Type> columnTypes)
             throws IOException
     {
-        try (ConnectorPageSource pageSource = createPageSource(
+        try (ConnectorPageSource pageSource = createPageSourceWithUtcTz(
```

Comment:
Rather than changing this for all tests, we should add a boolean flag `readWithUtcTimezone` to `assertRoundTrip` and have specific tests enabling that. This way we can test with UTC as well as default and see where exactly the override was needed.

Reply:
Now ParquetTester doesn't require any change after changing the parquet writer.